setup.py

```diff
@@ -7,7 +7,7 @@
 
 try:
     # Python 2 backwards compat
-    from __builtin__ import raw_input as input
+    from builtins import raw_input as input
```

Comment:
the above line is giving following error in my python3 setup,
```python
>>> from builtins import raw_input as input
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'raw_input' from 'builtins' (unknown location)
>>> 
```
you may have to change it to,
```python
from six.moves import input
```
this will work in both python 2.7 and 3.x (tested on python 2.7.16 and python 3.7.5)

Reply:
can you try this: 
    from builtins import input

no need alias, raw_input is renamed as input

Reply:
Thank you, I will do it!